babs/utils.py

```diff
@@ -16,6 +16,9 @@
 from datetime import datetime
 import re
 
+
+TYPE_SYSTEM_SUPPORTED = ['sge', 'slurm']
```

Comment:
This is a good idea! Do you mind moving this "constant" variable into `babs/constants.py` and importing it here? That python file holds several constant variables.